lib/browser_sniffer/version.rb

```diff
@@ -1,3 +1,3 @@
 class BrowserSniffer
-  VERSION = "1.2.2"
+  VERSION = "1.2.3"
```

Comment:
Given that https://github.com/Shopify/browser_sniffer/pull/34 added functionality in a backwards compatible way (adds net new public method, `ie11_actual?`), I think this bump should be to `1.3.0`

https://semver.org/
```suggestion
  VERSION = "1.3.0"
```